<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Widget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Farmer">
       <attribute name="title">
        <string>Farmer</string>
       </attribute>
       <widget class="QWidget" name="InputPart" native="true">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>0</y>
          <width>185</width>
          <height>307</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="InitChia">
           <property name="text">
            <string>Initialiser Chia</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="autoRepeatDelay">
            <number>301</number>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="ModeSombre">
           <property name="text">
            <string>Mode Sombre</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="ArretRelance">
           <property name="text">
            <string>Arret Puis Relance Chia</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="LogoLabel">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>0</y>
          <width>261</width>
          <height>301</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../../Desktop/Bitcoin512-195x195.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>331</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="LoadCaCert">
           <property name="text">
            <string>Chemin vers les dossiers certificat chia</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="CaDisplayPath">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>180</y>
          <width>331</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QPushButton" name="RunChia">
           <property name="text">
            <string>Lancement de Chia</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="TelegrafTab">
       <attribute name="title">
        <string>Telegraf</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>161</width>
          <height>271</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="TelegrafInputBox">
         <item row="0" column="0">
          <widget class="QRadioButton" name="LinuxRadio">
           <property name="text">
            <string>Linux</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="ChiaEnabled">
           <property name="text">
            <string>Chia Monitoring</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="TRexEnabled">
           <property name="text">
            <string>T-rex Monitoring</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="windowsRadio">
           <property name="text">
            <string>Windows</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="SmiEnabled">
           <property name="text">
            <string>Smi Monitoring</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="CustomProcessEnabled">
           <property name="text">
            <string>Custom process</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>639</x>
          <y>210</y>
          <width>191</width>
          <height>80</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="RunButtonLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="GenerateJinja">
           <property name="text">
            <string>GÃ©nerer le template</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>201</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="EsLayout">
         <item row="0" column="0">
          <widget class="QLineEdit" name="UsernameEs">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Usename</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="Password">
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="CustomName">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Host Name (optionel)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ChiaWidget" native="true">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>160</y>
          <width>209</width>
          <height>129</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>211</width>
           <height>131</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QPushButton" name="LoadKey">
            <property name="text">
             <string>Chemin de la clef ssl</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="LoadCrt">
            <property name="text">
             <string>Chemin du certificat ssl</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="LauncherId">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>launcher_id</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="SMIWidget" native="true">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>20</y>
          <width>261</width>
          <height>49</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>0</y>
           <width>251</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="SmiBinPath">
            <property name="text">
             <string>Chemin du binaire SMI</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="CustomProcessWidget" native="true">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>70</y>
          <width>171</width>
          <height>78</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_8">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>161</width>
           <height>71</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="CustomProcessLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="ProcessInput">
            <property name="alignment">
             <set>Qt::AlignJustify|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>process_1,process_2</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="APropos">
       <attribute name="title">
        <string>A propos</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
